doc/howtos/building-images.rst

```diff
@@ -230,17 +241,34 @@ The following instructions assume that swupd is not used.
 
 .. TODO: document how to create custom image recipes based on ostro-image.bbclass.
 
+.. _`ostro-image.bbclass`: https://github.com/ostroproject/meta-ostro/blob/master/meta-ostro/classes/ostro-image.bbclass
+
+Installing Additional Packages (without swupd)
+----------------------------------------------
+
+An image derived from ``ostro-image.bbclass`` without additional
+configuration is very minimal and establishes a core OS with components
+that must always be present on a device. All additional components beyond
+this minimal configuration must be added explicitly by setting
+``OSTRO_IMAGE_EXTRA_FEATURES`` or ``OSTRO_IMAGE_EXTRA_INSTALL`` adding them
+to the image when building without swupd.
 
-Installing Additional Packages without swupd
---------------------------------------------
+The ``ostro-os`` repo contains many layers and recipies that are not enabled
+but are available for your use. You can see these by using the commands:::
+
+   $ bitbake-layers show-recipes
+   $ bitbake-layers show-layers
 
 ``ostro-image.bbclass`` defines several image features which can be enabled
-to install additional sets of components. For example, to install debugging
+to install additional sets of pre-defined components. For example, to install debugging
 tools, compilers and development files for all components in the image, add::
 
     OSTRO_IMAGE_EXTRA_FEATURES += "tools-debug tools-develop dev-pkgs"
 
-Extend ``OSTRO_IMAGE_EXTRA_INSTALL`` to install additional individual packages,
+See your local copy of ``ostro-image.bbclass`` for more image feature options or
+you can view `ostro-image.bbclass`_ from the GitHub repository.
```

Comment:
s/the GitHub/the upstream GitHub

The link is good in that it points at the upstream repo, not the one that is part of the combined ostro-os repo. It's a subtle difference but I think it's worth pointing it out.